tcms/testcases/tests/test_models.py

```diff
@@ -28,6 +29,43 @@ def test_create_testcase_with_cyrillic_works_issue_1770(self):
         case.refresh_from_db()
         self.assertTrue(case.summary.endswith("кирилица"))
 
+class TestCaseCalculateExpectedDuration(TestCase):
+    """Test TestCase.expected_duration"""
+
+    def test_duration_is_calculated_correctly(self):
+        case_random_duration = TestCaseFactory()
+        setup_duration = case_random_duration.setup_duration
+        testing_duration = case_random_duration.testing_duration
+        expected = setup_duration + testing_duration
+
+        case_random_duration.save()
+
+        case_random_duration.refresh_from_db()
+        self.assertEqual(case_random_duration.expected_duration, expected)
+
+    def test_empty_setup_duration_is_calculated_correctly(self):
+        case_empty_setup_duration = TestCaseFactory(setup_duration = None)
```

Comment:
name this variable `test_case` to make the code more obvious. The test method name serves to describe what behavior is being exercised so variable names do not need to carry the same information.